README.md

```diff
@@ -1,70 +1,42 @@
-# Getting Started with Create React App
+# Project Name
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+>mathmagician .
 
-## Available Scripts
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- react
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+**To get this project locally.**
+## Run this command on your terminal
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
+```
+git clone https://github.com/businesskaniki/mathmagician.git
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```
+## Authors
 
-### `npm run build`
+👤 **Author1**
```

Comment:
- [ ] Please put your name here instead of `Author1` to make your README file look more professional.